src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionTanInputViewController.swift

```diff
@@ -109,8 +110,19 @@ extension ExposureSubmissionTanInputViewController: ExposureSubmissionNavigation
 
 	// MARK: - ENATanInputDelegate
 
-	func tanChanged(isValid: Bool) {
-		setButtonEnabled(enabled: isValid)
+	func tanChanged(isValid: Bool, checksumIsValid: Bool, isBlocked: Bool) {
+		setButtonEnabled(enabled: (isValid && checksumIsValid))
+		if (isValid && !checksumIsValid) {
+			errorLabel.text = AppStrings.ExposureSubmissionTanEntry.invalidError
+			errorLabel.isHidden = false
+		}
+		else if (isBlocked) {
```

Comment:
does the linter allow () in conditional statements?

Reply:
remove () around condition